src/D2L.Bmx/WriteHandler.cs

```diff
@@ -36,15 +36,23 @@ FileIniDataParser parser
 			output = "./" + output;
 		}
 		var credentialsFile = new SharedCredentialsFile( output );
-		var data = parser.ReadFile( credentialsFile.FilePath );
+		string credentialsFilePath = credentialsFile.FilePath;
```

Comment:
nit: nice to get rid of the `credentialsFile` variable if it's only used once to get the path
```csharp
string credentialsFilePath = new SharedCredentialsFile( output ).FilePath;
```